oauth2-server/src/main/java/com/clouway/oauth2/IssueNewTokenActivity.java

```diff
@@ -64,7 +64,12 @@ public Response execute(Client client, Identity identity, Set<String> scopes, Re
               .setSubject(identity.id())
               .setIssuedAt(instant.asDate())
               .setExpiration(new Date(accessToken.expirationTimestamp()))
-              .claim("identity", identityAsJson(identity).toString())
+              .claim("id", identity.id())
+              .claim("name", identity.name())
+              .claim("email", identity.email())
+              .claim("given_name", identity.givenName())
+              .claim("family_name", identity.familyName())
+              .claim("_private_claims", privateClaims(identity).toString())
```

Comment:
As I understand right now the custom claims that we have are namespace and isAdmin. Should I just add them as _namespace and _isAdmin (adding the underscore just to be sure not to get those collisions) or should I try to hack my way around the cycle in Identity.claims() and then add them as the names that they have in that Map?